x-pack/plugins/transform/public/app/sections/transform_management/components/action_reset/reset_action_name.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiToolTip } from '@elastic/eui';
+import { TransformState, TRANSFORM_STATE } from '../../../../../../common/constants';
+import { createCapabilityFailureMessage } from '../../../../lib/authorization';
+import { TransformListRow } from '../../../../common';
+
+export const resetActionNameText = i18n.translate(
+  'xpack.transform.transformList.resetActionNameText',
+  {
+    defaultMessage: 'Reset',
+  }
+);
+
+const transformCanNotBeReseted = (i: TransformListRow) =>
+  !([TRANSFORM_STATE.STOPPED, TRANSFORM_STATE.FAILED] as TransformState[]).includes(i.stats.state);
+
+export const isResetActionDisabled = (items: TransformListRow[], forceDisable: boolean) => {
+  const disabled = items.some(transformCanNotBeReseted);
+  return forceDisable === true || disabled;
+};
+
+export interface ResetActionNameProps {
+  canResetTransform: boolean;
+  disabled: boolean;
+  isBulkAction: boolean;
+}
+
+export const ResetActionName: FC<ResetActionNameProps> = ({
+  canResetTransform,
+  disabled,
+  isBulkAction,
+}) => {
+  const bulkResetButtonDisabledText = i18n.translate(
+    'xpack.transform.transformList.resetBulkActionDisabledToolTipContent',
+    {
+      defaultMessage: 'One or more selected transforms must be stopped in order to be reseted.',
```

Comment:
Typo `reseted`. Maybe `One or more selected transforms must be stopped in order to be reset` ?

Reply:
Maybe we can make it a bit shorter and it might also sound better.
```suggestion
      defaultMessage: 'One or more selected transforms must be stopped to be reset.',
```